src/Compilers/CSharp/Test/Symbol/Symbols/Source/FileModifierTests.cs

```diff
@@ -3603,6 +3606,7 @@
         var sourceType = comp.GetTypeByMetadataName("<>FE3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855__C`1");
         Assert.Equal(sourceMember, sourceType);
         Assert.Null(comp.GetTypeByMetadataName("<>FE3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855__C"));
+        Assert.Null(comp.GetTypeByMetadataName("C`1"));
```

Comment:
>Assert.Null(comp.GetTypeByMetadataName("C`1")); [](http://example.com/codeflow?start=8&length=47)

Please also assert:
```
Assert.Null(comp.GetTypeByMetadataName("C"));
```

Reply:
I am not following the logic of why you requested such assert exactly in this place, since there is only a generic version of type `C` in the scope. So I added the assert only here. LMK if you meant to add an assert of `C` wherever there is an assert of ``C`1`` and vise versa

Reply:
> I am not following the logic of why you requested such assert exactly in this place, since there is only a generic version of type C in the scope. 

To make sure we are getting expected result for the scenario.